src/main/java/bowling/view/Printer.java

```diff
@@ -0,0 +1,39 @@
+package bowling.view;
+
+import bowling.domain.Frame;
+import bowling.domain.FrameSet;
+import bowling.domain.GameResult;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Printer {
+    public static void showResult(FrameSet frameSet) {
+        showHead(frameSet);
+        showBody(frameSet);
+        System.out.println();
+    }
+
+    private static void showHead(FrameSet frameSet) {
+        System.out.print("| NAME |");
+        for (Frame frame : frameSet.getFrames()) {
+            System.out.print(String.format("  %02d  |", frame.getStage()));
+        }
+        System.out.println();
+    }
+
+    private static void showBody(FrameSet frameSet) {
+        System.out.print(String.format("| %4s |", frameSet.getBowler()));
+
+        for (Frame frame : frameSet.getFrames()) {
+            List<GameResult> results = frame.getResults();
+            System.out.printf(
+                    String.format("  %-3s |",
+                            results.stream()
+                                    .map(result -> String.valueOf(result))
```

Comment:
메서드 레퍼런스 사용으로 조금 더 간결 해질 수 있겠어요! 😄 
```suggestion
                                    .map(String::valueOf)
```

Reply:
네 수정하도록하겠습니다~ :)